### PR TITLE
Update docs for skipping 0.31

### PR DESCRIPTION
**Pull request by doveye**
https://github.com/eclipse-openj9/openj9-docs/issues/921
Renamed 0.31 What's New topic to 0.32, updated text accordingly, made some other improvements.
Updated releases table in the supported environments topic.

Signed-off-by: Esther <doveye@uk.ibm.com>

### CODE REVIEW

**Comment by doveye** [comment]
Jenkins doc stage

**Comment by doveye** [comment]
@pshipton could you please check and merge if ok, thanks.

File: docs/openj9_support.md [hunk 1]
```diff
@@ -43,15 +43,14 @@ columns will be removed over time.
...
+| 0.33.0         | Jul 2022 (1)  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |                                                                                          |
+| 0.34.0         | Sep 2022 (1)  | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-times:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> (2) |
```
Comment (pshipton) [inline docs/openj9_support.md:53]: Only Java 19 should be supported in 0.34
8,11, 17, 18 should have a red x.
https://eclipse-openj9.github.io/openj9-docs-staging/922/openj9_support/

**Comment by doveye** [comment]
Jenkins doc stage

File: docs/openj9_support.md (hunk 1 above)
Comment (doveye) [reply docs/openj9_support.md]: Of course - I knew there were too many ticks in the table! Thanks for spotting that, it's now fixed.

**Review by pshipton — APPROVED** [review]

**Pull request merged**